src/current/v23.2/ui-queues-dashboard.md

```diff
@@ -180,6 +167,25 @@ Metric | Description
 Successful Actions | The number of replicas successfully processed by the time series maintenance queue, as tracked by the `queue.tsmaintenance.process.success` metric.
 Pending Actions | The number of pending replicas in the time series maintenance queue, as tracked by the `queue.tsmaintenance.pending` metric.
 
+## MVCC GC Queue
```

Comment:
Can we add a link to describe the concepts of MVCC and PTS records? Here's the closest do i've found:

https://www.cockroachlabs.com/docs/stable/architecture/storage-layer#garbage-collection

Reply:
Updated. Added link.